src/graph_compute/bsp_scheduling.h

```diff
@@ -14,14 +14,18 @@ static void init_scheduling(vertex_t * const nodes, const vid_t cntNodes,
   // no-op
 }
 
-static void execute_round(const int round, vertex_t * const nodes, const vid_t cntNodes,
+static void execute_round(const int numRounds, vertex_t * const nodes,
```

Comment:
Generally, the idea is to not merge pull requests until all comments are resolved, so that the master branch always has a working, satisfactory version of the code. That would mean advancing `pr_rollback` by pushing to it, and rebasing other branches as necessary.

This is called the feature-branch workflow, see here for details: https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow